src/java.base/share/classes/java/util/Locale.java

```diff
@@ -1826,13 +1826,15 @@ public static String caseFoldLanguageTag(String languageTag) {
      *     loc.getExtension('x'); // returns "urp"
      * }
      *
-     * <li>When the languageTag argument contains an extlang subtag,
-     * the first such subtag is used as the language, and the primary
+     * <li>{@code languageTag} may contain up to three extlang subtags.
+     * For such occurrences, the first extlang subtag is used as the language, and the primary
      * language subtag and other extlang subtags are ignored:
      *
      * {@snippet lang=java :
      *     Locale.forLanguageTag("ar-aao").getLanguage(); // returns "aao"
      *     Locale.forLanguageTag("en-abc-def-us").toString(); // returns "abc_US"
+     *     Locale.forLanguageTag("zh-yue-gan-cmn-czh-CN").toString();
+     *     // returns "yue"; the rest of the tag is considered ill-formed
```

Comment:
I'd rather not use "ill-formed" here, or change the wording, as "the rest of the tag" implies subtags other than "yue" which include "zh." If this "zh" is considered ill-formed, following "yue" should be discarded according to the prior definition.

Reply:
OK, I'd like to keep "ill-formed" because it allows the example to have meaning for both `[for/set]LanguageTag`.
I updated the wording so that it is clear that "czh", (the fourth extlang) and subsequent subtags are what is considered ill-formed.